modules/dublincore/src/test/java/org/opencastproject/metadata/dublincore/DublinCoreXmlFormatTest.java

```diff
@@ -97,4 +100,49 @@ public void readFromNode() throws Exception {
     assertNotNull(dc);
     assertEquals("10.0000/5820", DublinCores.mkOpencastEpisode(dc).getDcIdentifier().get());
   }
+
+  @Test
+  public void readWithEmptyAndMergeTest() throws Exception {
+    String emtpy = "";
+    String delimiter = "|";
+    String irishTitle = "Talamh agus Fásra: Príomh-imreoirí ar an Radharc Aeráide";
+    String irishLang = "ga";
+    String germanLang = "de";
+    String fooId = "197011";
+    EName eNamepropertyFooId = new EName("http://foo.org/metadata", "id");
+    boolean marshalEmpty = true;
+
+    DublinCoreCatalog catalogWithEmpty = DublinCoreXmlFormat
+            .read(IoSupport.classPathResourceAsFile("/dublincore.xml").get(), marshalEmpty);
+    DublinCoreCatalog catalogNoEmpty = DublinCoreXmlFormat
+            .read(IoSupport.classPathResourceAsFile("/dublincore.xml").get(), !marshalEmpty);
+
+    // Loading with empty and not empty
+    assertEquals(emtpy,catalogWithEmpty.getFirst(DublinCore.PROPERTY_SPATIAL));
+    assertNull(catalogNoEmpty.getFirst(DublinCore.PROPERTY_SPATIAL));
+
+    DublinCoreCatalog mergedEmptyNoEmpty = DublinCoreXmlFormat.merge(catalogWithEmpty, catalogNoEmpty);
+
+    // Extra empty values have no effect on merge, original is not affected
+    assertEquals(catalogNoEmpty, mergedEmptyNoEmpty);
+    assertEquals(irishTitle, catalogNoEmpty.getAsText(DublinCore.PROPERTY_TITLE, irishLang, delimiter));
+    assertNull(catalogNoEmpty.getFirst(DublinCore.PROPERTY_SPATIAL));
+
+
```

Comment:
I think two blank lines violate our styling policy? (Okay, I'm done with the nitpicking now...)